pyscriptjs/src/runtime.ts

```diff
@@ -0,0 +1,101 @@
+import type { loadPyodide } from 'pyodide';
+
+// The current release doesn't export `PyodideInterface` type
+export type PyodideInterface = Awaited<ReturnType<typeof loadPyodide>>;
+export type RuntimeInterpreter = PyodideInterface | null;
+
+/*
+Runtime class is a super class that all different runtimes must respect
+and adhere to.
+
+Currently, the only runtime available is Pyodide as indicated by the
```

Comment:
It's ok as first step.... we should change this soon to reflect that pyodide is the default rather the only one

Reply:
done, removing a line about `design is based on Pyodide`